src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Methodologies/MethodologyService.cs

```diff
@@ -323,13 +325,18 @@ await _context.Entry(loadedMethodologyVersion.ScheduledWithRelease)
                 return methodologyVersionToUpdate;
             }
 
+            var newSlug = SlugFromTitle(request.Title);
+
             return await _userService.CheckCanUpdateMethodologyVersion(methodologyVersionToUpdate)
-                // Check that the Methodology will have a unique slug.  It is possible to have a clash in the case where
-                // another Methodology has previously set its AlternativeTitle (and Slug) to something specific and then
-                // this Methodology attempts to set its AlternativeTitle (and Slug) to the same value.  Whilst an
-                // unlikely scenario, it's entirely possible.
-                .OnSuccessDo(methodologyVersion =>
-                    ValidateMethodologySlugUniqueForUpdate(methodologyVersion, request.Slug))
+                .OnSuccessDo(async _ => {
+                    if (newSlug != methodologyVersionToUpdate.Methodology.Slug)
+                    {
+                        // Only need to validate if the slug is changing
+                        return await ValidateMethodologySlugUnique(newSlug);
```

Comment:
I think you could move the `newSlug != {existing}.Slug` check into this method. I appreciate we want to avoid making a call to the repository if we don't have to, but you can still do that inside the method. At the moment it feels like we're doing half the validation inside the method and half outside.

I also feel like this could be a bit more readable - it's surprising that a method named `ValidateFoo` should return an ActionResult or a Unit. I'd usually have expected it to return an object representing validation errors, or just a boolean flag for valid or not. Returning the ActionResult feels like we're merging our HTTP logic with our validation logic, and doing two things in one method.